smartcar/helpers.py

```diff
@@ -89,6 +89,8 @@ def format_flag_query(flags: dict) -> str:
     flags_str = ""
 
     for flag in flags.keys():
```

Comment:
```suggestion
    for flag_name, flag_value in flags.items()
```
Thoughts? So you don't have to keep doing `flags[flag]`